src/system/UsersModule/Helper/MailHelper.php

```diff
@@ -113,7 +113,7 @@ public function createAndSendRegistrationMail(UserEntity $userEntity, $userNotif
             $notificationEmail = $this->variableApi->get('ZikulaUsersModule', UsersConstant::MODVAR_REGISTRATION_ADMIN_NOTIFICATION_EMAIL, '');
             if (!empty($notificationEmail)) {
                 $authMapping = $this->authenticationMappingRepository->getByZikulaId($userEntity->getUid());
-                $rendererArgs['isVerified'] = $authMapping->isVerifiedEmail();
+                $rendererArgs['isVerified'] = null === $authMapping || $authMapping->isVerifiedEmail();
```

Comment:
sorry, I'm not reading the code carefully enough. hard for me to multitask. You're right - email verification is part of ZAuth.